src/elements/content-sidebar/ContentSidebar.scss

```diff
@@ -30,10 +30,6 @@ $sidebarIncreasedWidth: $sidebarTabsWidth + $sidebarContentIncreasedWidth;
     &.bcs-is-open {
         .bcs-loading {
             width: $sidebarWidth;
-
-            &.bcs-is-wider {
-                width: $sidebarIncreasedWidth;
-            }
```

Comment:
I didn't mean this needs to be removed completely. I guess we still need to apply the increased width to the `.bcs-loading` element. This could be added "inside" `.be` selector
```
.be {
    &.bcs {
        ...
        &.bcs-is-wider.bcs-is-open .bcs-loading {
            width: $sidebarIncreasedWidth;
        }
    }
```